example/ios/SampleApp/AppDelegate.swift

```diff
@@ -9,6 +9,7 @@ import UserNotifications
 import FirebaseMessaging
 import FirebaseCore
 import CioMessagingPushFCM
+import CioFirebaseWrapper
```

Comment:
Wouldn't this be a breaking change for FCM customers? Do we still want to keep `feat` prefix (minor release) instead of major release? 🤔